
The resulting macros hacks are to ugly. If anyone needs this optimization
he is free to fix the ugly C hacks (damn when start people learning s.th.
other than this 70's C style ...)

  - Rene Rebe <rene@exactcode.de>

--- gmp-4.1.2/gmp-impl.h.orig	2004-05-05 03:37:08.000000000 +0200
+++ gmp-4.1.2/gmp-impl.h	2004-05-05 03:37:30.000000000 +0200
@@ -577,7 +577,7 @@
    still use this, if we cared to differentiate locals and globals.  */
 
 #if HAVE_HOST_CPU_FAMILY_x86 && __GMP_GNUC_PREREQ (2,96) && ! defined (PIC)
-#define USE_LEADING_REGPARM 1
+#define USE_LEADING_REGPARM 0 /* 1 */
 #else
 #define USE_LEADING_REGPARM 0
 #endif
